README.md

```diff
@@ -11,6 +13,46 @@ npm install --save analytics-client
 
 ## Usage
 
+Tracking a page view. The following reports an event named `[EFP] Page View` with default properties.
+
+```typescript
+import { createClient, WebTracker, createWebTracker } from 'analytics-client';
+
+const client = createClient({
+    endpoint: 'data.balena-cloud.com', // use 'data.balena-staging.com' for testing the integration
+    projectName: 'balena-project', // unique identifier that analytics-backend expects
+    componentName: 'etcher-featured-project', // short unique name of component
+    componentVersion: '0.52', // automated version reporting of component is beneficial
```

Comment:
What does it mean by automated version reporting? I assume you mean something like `require('./package.json').version` but that's really not conveyed by this comment

Reply:
A good observation. I believe we're mostly interested in referencing the state/version of the tracking implementation, and that would come with the component version by default.

Reply:
> `require('./package.json').version`

Oh this is brilliant @Page-. I didn't know one can get the version like this. I'll update the example with it.